src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.constants :as constants]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view :as actions]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.style :as style]
+            [utils.address :as utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- parse-accounts
```

Comment:
Could it be a layer-3 subscription instead? More specifically, the result of the sub `:wallet/keypairs` is not ideal because it requires further processing in the view, so in theory, we could have another sub and use that instead of `:wallet/keypairs`. Once that's done, we would end up with a testable piece of layer-3 sub instead of this `parse-accounts` function.

Reply:
Sure thing! happy to make this a layer-3 sub 🙌